packages/office-ui-fabric-react/src/components/Icon/Icon.scss

```diff
@@ -0,0 +1,12 @@
+
+.ms-Icon-ImageContainer {
```

Comment:
Minor change: We follow the [SUIT CSS naming conventions](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md) which would call for this class to be `ms-Icon-imageContainer`